modin/pandas/test/test_series.py

```diff
@@ -1533,12 +1537,17 @@ def test_dropna_inplace(data):
     df_equals(modin_series, pandas_series)
 
 
-@pytest.mark.xfail_backends(
-    ["BaseOnPython"], reason="Empty Series has a missmatched from Pandas dtype."
-)
 def test_dtype_empty():
     modin_series, pandas_series = pd.Series(), pandas.Series()
-    assert modin_series.dtype == pandas_series.dtype
+    try:
+        assert modin_series.dtype == pandas_series.dtype
+    except Exception as e:
+        # TODO: Remove this try-except when default  `dtype` of empty Series will be `object` in pandas (see #3142).
+        if get_current_backend() == "BaseOnPython":
+            assert modin_series.dtype == np.object
+            assert pandas_series.dtype == np.float64
+        else:
+            raise e
```

Comment:
I would suggest extracting this check into a separate function to not copy-paste in tests